tensorvision/utils.py

```diff
@@ -8,12 +8,12 @@
 import logging
 import numpy as np
 import os
+import sys
 import time
 
 from datetime import datetime
 
 # https://github.com/tensorflow/tensorflow/issues/2034#issuecomment-220820070
```

Comment:
Original numpy got imported twice (line 9, line 16). Is this really necessary?  If I understand the comment correctly important is only, that numpy gets imported before tensorflow. Am I right with this?